daemon/src/routes_taker.rs

```diff
@@ -65,10 +65,10 @@ pub struct CfdOrderRequest {
 #[rocket::post("/cfd", data = "<cfd_order_request>")]
 pub async fn post_order_request(
     cfd_order_request: Json<CfdOrderRequest>,
-    cfd_actor_inbox: &State<Address<TakerCfdActor>>,
+    cfd_actor_inbox: &State<Address<Actor>>,
```

Comment:
Can we refer to this via the module name please?
```suggestion
    cfd_actor_inbox: &State<Address<taker_cfd::Actor>>,
```

Reply:
sure!